presto-docs/src/main/sphinx/release/release-0.263.rst

```diff
@@ -0,0 +1,23 @@
+=============
+Release 0.263
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a correctness bug in distinct aggregations when combined with a filtering mask containing null values.
+* Improve spilling CPU and memory efficiency for ``DISTINCT`` and ``ORDER BY`` aggregations.
+* Add a new session property ``heap_dump_on_exceeded_memory_limit_enabled`` to enable heapdump on exceeded memory failures. The heapdump file directory can be provided using ``exceeded_memory_limit_heap_dump_file_directory`` session property.
+* Add key-based sampling feature that samples tables at query time. This feature can be enabled using session property ``key_based_sampling_enabled=true``. Additionally, default sampling rate and sampling function name can be overridden by sessions properties ``key_based_sampling_percentage`` and ``key_based_sampling_function`` respectively.
+* Support distinct user defined types with ``CREATE TYPE`` syntax.
```

Comment:
Is this a new language feature ? If yes, we should create documentation and link to the documentation.